handbook/engineering/roles.md

```diff
@@ -1,55 +1,61 @@
 # Roles on the Engineering team
 
-This page lists the roles and responsibilities of the Engineering team.
+This page lists the roles on the Engineering team and some of the responsibilites for each of the roles. 
 
 ## Software Engineer
 
-### Responsibilities
-
-- Write (the first pass) or update the documentation for features you create or change.
-- TODO
-
-### Tech Lead
-
-<!-- TODO(nicksnyder): this was taken from the old docs -->
-
-A Software Engineer who is a Tech Lead has the following additional responsibilities.
-
-The Tech Lead is responsible for making sure the following two statements are true:
+Sofware Engineers are individual contributors who build software to satisfy the needs of the business.
 
-> The issues for the current release milestone describe the problem/solution.
->
-> The issues for the current release milestone are completed (closed) before the release.
+### Responsibilities
 
-The following sections give more details.
+- Collaborate with your direct manager to identify projects that align your skills and interests with the needs of the business.
+- Write [RFCs](rfcs/index.md) to recommend solutions to product and engineering problems.
+- Commit to a reasonable amount of work that you are going to get done for each [monthly release](releases/index.md), and then reliably get it done.
+- Create and maintain high quality code, tests, and documentation.
+- Help your teammates by empathetically [reviewing code](code_reviews.md) and [RFCs](rfcs/index.md).
+- Interview candidates and suggest improvements to our hiring process.
 
-#### The issues for each release milestone are completed (closed) before the release.
+## Software Engineer Manager 
```

Comment:
Do you have a preference for this term compared to the shorter "Engineering Manager"?

Reply:
I had not given any thought to this. Engineering Manager is more standard and shorter so I changed this.